metrics-test.py

```
@@ -9,141 +9,141 @@
+    macro = f1_scoreall_true, pred, average="macro", zero_division=0
+    micro = f1_scoreall_true, pred, average="micro", zero_division=0
+    subset_acc = accuracy_scoreall_true, pred
     return macro, micro, subset_acc
 
 # 7) Report default and tuned thresholds
-m05, mi05, a05 = evaluate(0.50)
-ths = np.linspace(0.05, 0.6, 12)
-scores = [evaluate(t) for t in ths]
-best_idx = int(np.argmax([m for (m, _, _) in scores]))
-best_th = float(ths[best_idx])
+m05, mi05, a05 = evaluate0.50
```

Ruff (0.12.2)

SyntaxError: Expected a statement

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

No newline at end of file

(W292)